### ISSUE TITLE
Smoking findings

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9785]
**Preferred term label:**
Tobacco use history

**Synonyms**
Smoker

**Definition (free text, please give PubMed ID)**
HPO (and we) have two findings for triggering or exacerbation by smoke: HP:0025223 and HP:0032500.  We should consider consolidating those.  We should also have "Tobacco use history", e.g., for someone with a suspected MI, this is a major risk factor but not what we'd consider a trigger in the proximate sense.  We should also consider a finding for triggering by a flame without smoke, e.g., by the nitrogen dioxide produced by any flame in air as discussed in my WSJ article mirrored at https://segal.org/gas/.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
adding Tobacco use history HP:0430133
I am hoping the next release of phenopackets will provide a better way of recording this. It will be out of scope for the HPO to have lots of details as to how many pack years etc.
